app/controllers/systems_controller.rb

```diff
@@ -12,6 +12,13 @@ def edit
   def update
     @system = System.find(params[:id])
     if @system.update(system_params)
+      @system.units.each do |unit|
+        unless ( current_user == @system.gm ||
```

Comment:
Don't use parentheses around the condition of an `unless`.<br>Space inside parentheses detected.